File: Selenium/Project/Activity5.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.*;

public class Activity5 {

    WebDriver driver = new FirefoxDriver();

    @BeforeMethod
    public void openBrowser() throws InterruptedException {
        WebDriverManager.firefoxdriver().setup();
        driver.get("https://alchemy.hguy.co/jobs/");
        driver.manage().window().maximize();
        Thread.sleep(1000);
    }

    @Test
    public void JobPages() throws InterruptedException {
        WebElement JobsPage= driver.findElement(By.linkText("Jobs"));
        JobsPage.click();
        Thread.sleep(1000);

    }
    @AfterMethod
    public void verifyTitle(){
        WebElement title=driver.findElement(By.xpath("//title"));
        if(title.getAttribute("innerHTML").equals("Jobs – Alchemy Jobs"))
        {
            System.out.println("Title is :" +title.getAttribute("innerHTML"));
            driver.quit();
        }
        else{
            System.out.println("Title is not equals to:" +title.getAttribute("innerHTML"));
        }
    }



    @AfterTest
    public void closeBrowser(){
        driver.quit();

    }

}

File: Selenium/Project/Activity5.py
import pytest
import time

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())
# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # Open the browser to the URL
    driver.get("https://alchemy.hguy.co/jobs")
    driver.maximize_window()
    print("Page Title is ", driver.title)
    time.sleep(8)

    #click on job
    JobsPage=driver.find_element(By.LINK_TEXT,"Jobs")
    JobsPage.click()
    time.sleep(2)
    #verify title of Jobs page
    title=driver.find_element(By.XPATH,"//title")
    actualtitle="Jobs – Alchemy Jobs"
    if title.get_attribute("innerHTML")==actualtitle:
        print("\nTitle Matched")
    else:
        print("Title is not Matched")